plugins/woocommerce/src/Internal/Admin/Marketplace.php

```diff
@@ -71,4 +78,59 @@ public function enqueue_scripts( $hook_suffix ) {
 		// Enqueue WordPress updates script to enable plugin and theme installs and updates.
 		wp_enqueue_script( 'updates' );
 	}
+
+	/**
+	 * Add a Woo Marketplace link to the plugin install action links.
+	 * @param array $tabs Plugins list tabs.
+	 * @return array
+	 */
+	public function add_woo_plugin_install_action_link( $tabs ) {
+		$tabs[self::MARKETPLACE_TAB_SLUG] = _x( 'Woo', 'Plugin Installer', 'wccom' );
```

Comment:
Does the brand name need to be translated?

Reply:
True, updated it now.